toolflow/scala/src/main/scala/tapasco/activity/hls/VivadoHighLevelSynthesis.scala

```diff
@@ -77,11 +77,18 @@ private object VivadoHighLevelSynthesis extends HighLevelSynthesizer {
       logger.debug("Vivado HLS finished with exit code %d".format(vivadoRet))
       vivadoRet match {
         case 0 =>
-          logger.info("Vivado HLS finished successfully for {}", runName)
-          logger.trace("performing additional steps for {}", runName)
-          performAdditionalSteps(k, t)
-          logger.trace("additional steps for {} finished, copying zip", runName)
-          Success(HighLevelSynthesizerLog(logfile), copyZip(k, t).get)
+          val cp = copyZip(k, t)
+          if(cp.isEmpty) {
+            logger.error("Vivado HLS failed for {}", k.name)
```

Comment:
Sorry for the nitpicking, but shouldn't it be `runName` instead of `k.name`, because `runName` also contains information about the platform and architecture? It could be confusing, if HLS fails for a kernel on one platform and succeeds on another.